mdnDocsOverrides.json

```
@@ -375,14 +404,22 @@
   "class-syntax": [
     "Web/JavaScript/Reference/Classes"
   ],
-  "clear-site-data": [],
+  "clear-site-data": [
+    "Web/HTTP/Headers/Clear-Site-Data"
+  ],
   "clip-path": [
     "Web/CSS/clip-path",
     "Web/SVG/Attribute/clip-path"
   ],
-  "clip-path-animatable": [],
-  "clip-path-boxes": [],
-  "clipboard-custom-format": [],
+  "clip-path-animatable": [
+    "Web/CSS/clip-path"
+  ],
+  "clip-path-boxes": [
+    "Web/CSS/clip-path#geometry-box"
+  ],
+  "clipboard-custom-format": [
+    "Web/API/ClipboardItem"
+  ],
```

There's a page on MDN that discusses custom format support. It's not exactly the right page, but it might be useful to people to add it:

```suggestion
  "clipboard-custom-format": [
    "Web/API/ClipboardItem",
    "Web/API/ClipboardItem/supports_static#type"
  ],
```